monitoring/monitor.py

```diff
@@ -0,0 +1,66 @@
+import os
```

Comment:
Same with this file, what are we using it for? Is it just dev time testing/monitoring? If it's useful let's add it to the README so others can understand it.